tests/system/providers/amazon/aws/example_ecs.py

```diff
@@ -67,6 +66,15 @@ def get_region():
     return boto3.session.Session().region_name
 
 
+@task(trigger_rule=TriggerRule.ALL_DONE)
+def clean_logs(group_name: str):
+    client = boto3.client("logs")
+    # A bit brutal to delete the whole group, I know,
+    # but we don't have the access to the arn of the task which is used in the stream name
+    # and also those logs just contain "hello world", which is not very interesting.
+    client.delete_log_group(logGroupName=group_name)
```

Comment:
this is going to fail if the group does not exist, so in a way it makes sure the log configuration stays correct.